wurlitzer.py

```diff
@@ -155,17 +155,25 @@ def forwarder():
             """Forward bytes on a pipe to stream messages"""
             done = False
             draining = False
+            flush_interval = 0
             while pipes:
-                # flush libc's buffers before calling select
-                libc.fflush(c_stdout_p)
-                libc.fflush(c_stderr_p)
-                r, w, x = select.select(pipes, [], [], self.flush_interval)
-                if not r:
+                r, w, x = select.select(pipes, [], [], flush_interval)
```

Comment:
Why did you change self.flush_interval to flush_interval?

Reply:
Because now it may be self._flush_interval or 0 depending on the previous state.